tweetscreation.py

```diff
@@ -0,0 +1,20 @@
+from feedparser import parse
+from sys import argv
+
+tweets_file = "tweets.txt"
```

Comment:
probably a good idea to pick this up from settings.

Reply:
Pardon, I don't understand. Settings as in config.json?